### PR TITLE
tools: Add scripts to check all 'bugrefs' for status on bugzilla; various cleanup

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: tools/check_bugrefs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+import configparser
+import os.path
+import requests
+import sys
+
+config = configparser.ConfigParser()
+config.read(os.path.expanduser('/home/okurz/.oscrc'))
```
Comment (coolo) [inline tools/check_bugrefs]: that might be suprising to you, but most can't read that file

Comment (okurz) [inline tools/check_bugrefs]: ah, ok. I better include my password directly then ;-) Sorry, there is a reason I used 'expanduser'. I just didn't make proper use of it.

**Comment by okurz** [comment]
Local verification run: http://lord.arch/tests/5382

please wait for result of test job if it actually passes or not.

File: tests/x11/khelpcenter.pm [hunk 2]
```diff
@@ -8,24 +8,17 @@
...
-    if (get_var("LIVETEST")) {
-        assert_screen 'test-khelpcenter-1', 15;
-    }
-    else {
-        assert_screen 'test-khelpcenter-1', 3;
-    }
-    send_key "alt-f4";
-    sleep 2;
+    assert_screen 'test-khelpcenter-1';
+    wait_screen_change { send_key 'alt-f4' };
```
Comment (coolo) [inline tests/x11/khelpcenter.pm]: the wait_screen_change is pointless as the test ends here anyway - and afterwards comes an assert_screen 'generic-desktop'.

Comment (okurz) [inline tests/x11/khelpcenter.pm]: it's not pointless as it documents the intention. The screen is expected to change when I press alt-f4.

File: lib/x11test.pm [hunk 3]
```diff
@@ -56,5 +56,22 @@ sub prepare_sle_classic {
...
+}
+
+sub test_terminal {
+    my ($self, $name) = @_;
+    mouse_hide(1);
+    x11_start_program($name);
+    assert_screen $name;
+    $self->enter_test_text($name);
+    assert_screen "test-$name-1";
+    wait_screen_change { send_key 'alt-f4' };
```
Comment (okurz) [inline lib/x11test.pm]: @coolo so I should delete that `wait_screen_change`, too, and implicitly assume it's ensured from the outside that the corresponding action already finished before asserting anything else in the next step?

File: tests/x11/gedit.pm [hunk 4]
```diff
@@ -8,29 +8,21 @@
...
-    assert_screen 'test-gedit-1', 3;
-    sleep 2;
-    send_key "alt-f4";
-    sleep 2;
-    send_key "alt-w";
-    sleep 2;
+    $self->enter_test_text('gedit');
+    assert_screen 'test-gedit-1';
+    wait_screen_change { send_key 'alt-f4' };
+    wait_screen_change { send_key 'alt-w' };
```
Comment (coolo) [inline tests/x11/gedit.pm]: here definitely. Last action doesn't need any waiting - the waiting is implicit in x11test

**Pull request merged**